dev/com.ibm.ws.microprofile.openapi/src/com/ibm/ws/microprofile/openapi/impl/validation/HeaderValidator.java

```diff
@@ -36,14 +39,39 @@ private HeaderValidator() {}
     public void validate(ValidationHelper helper, Context context, String key, Header t) {
 
         if (t != null) {
+
             String reference = t.getRef();
 
             if (reference != null && !reference.isEmpty()) {
                 ValidatorUtils.referenceValidatorHelper(reference, t, helper, context, key);
-                return;
```

Comment:
this line is needed to exit the method, since the item is a reference. Otherwise it'll incorrectly check for the existence of one of schema and content.